Whiskey/Tasks/Set-WhiskeyProGetAsset.ps1

```diff
@@ -0,0 +1,97 @@
+
+
+function Set-WhiskeyProGetAsset {
+    <#
+        .SYNOPSIS
+        Adds and Updates assets to the ProGet asset manager. 
+
+        .DESCRIPTION
+        The `Set-WhiskeyProGetAsset` adds assets to ProGet. A root directory needs to be 
+        created in ProGet using the `New-ProGetFeed` function in ProGetAutomation with Type `Asset`.
+        
+        * `Path` (Mandatory): The relative paths to the files/directories to upload to ProGet. Paths should be relative to the whiskey.yml file they were taken from.

```

Comment:
Put a section header before the list of properties, e.g.

```
## Properties
```

Reply:
Done